### DISCUSSION TITLE
Cannot start nuxt:  Cannot use 'import.meta' outside a module

### DISCUSSION THREAD

**Discussion by shayandrg** [nuxt/nuxt#24507] [Questions]
**Environment**
Operating System: Ubuntu 22.04.3 LTS
Node Version: v20.8.1
Nuxt Version: 3.8.2
CLI Version: 3.10.0
Nitro Version: 2.8.0
Package Manager: npm@10.1.0
jiti: 1.21.0
telemetry: 2.5.3

**Describe the bug**
i tried to create nuxt module with default template as (nuxt.new) says and i done everything like nuxt module author document said but i got this error when starting nuxt with npm run dev:

![image](https://github.com/nuxt/nuxt/assets/45017318/5278ccdb-1b46-4114-bca2-2d3932b5e430)

i tried to clear npm local and global cache and remove parent folders node_modules and clear and re-installing more than 3 times everything and changing nuxt/kit version to previous but nothing happend.

im not new to nuxt module and i developed some modules for nuxt 2 in the past but I never had such an experience

what should i do?

**Comment by TheAlexLichter** [comment]
As written in https://github.com/nuxt/nuxt/issues/24185#issuecomment-1809897871, check your module and what you actually import :)

**Reply by shayandrg** [reply]
i just import defineNuxtModule bro

![image](https://github.com/nuxt/nuxt/assets/45017318/89a22a7a-69d7-4032-a06c-d389ba6c1691)

**Comment by matthew-dean** [comment]
Can confirm this is a Nuxt bug. It's a proper `.mjs` file being imported into `nuxt.config.ts` but the `import.meta` call within the `.mjs` file is not correctly interpreted and throws this error.

**Reply by TheAlexLichter** [reply]
A full issue is appreciated then 👍🏻

**Comment by LeonardoRick** [comment]
Is there any issue opened already?

**Reply by TheAlexLichter** [reply]
Doesn't look like, feel free to open one.
